pkg/v1/progress.go

```diff
@@ -0,0 +1,7 @@
+package v1
```

Comment:
Can you add a copyright header here?

Reply:
Sure. Feels strange to add a google copyright to it when I wrote it, but doesn't matter all that much.